src/pages/junhyeok/List/List.js

```diff
@@ -1,6 +1,46 @@
-import React from 'react';
-function List(){
+import React, { useState, useEffect } from 'react';
+import CoffeeCard from '../Component/CoffeeCard';
+import { Nav } from '../Component/NavAndBottom';
 
+import './List.scss';
+
+function List() {
+  useEffect(() => {
+    fetch('http://localhost:3000/data/junhyeok/data.json')
+      .then((res) => res.json())
+      .then((res) => setMockdata(res));
+  }, []);
+
+  let [mockdata, setMockdata] = useState([]);
+
+  return (
+    <div className="List">
+      <Nav />
+      {/* 콜드브루 */}
+      {Coffee(mockdata, '콜드 브루 커피', 'cold')}
+      {/* 아메리카노 */}
+      {Coffee(mockdata, '아메리카노', 'shot')}
+    </div>
+  );
+}
+
+//  반복요소 줄이기
+function Coffee(mockdata, name, kind) {
```

Comment:
```suggestion
      <Coffee data=mockdata name="콜드 브루 커피" cate="cold"/>
      {/* 아메리카노 */}
      <Coffee mockdata=mockdata name="아메리카노" cate="shot" />
    </div>
  );
}

//  반복요소 줄이기
function Coffee( { mockdata, name, cate } ) {
```

리액트 컴포넌트라면 기존 자바스크립트 함수 형태보다는 이렇게 사용하시는게 좋을 것 같습니다!

Reply:
넵! 변경하겠습니다